### DISCUSSION TITLE
Monorepo

### DISCUSSION THREAD

**Discussion by danielminhnguyen** [infinitered/ignite#1862] [General]
Any here success on creating a monorepo, nextjs with this boiler plate?

**Comment by jamonholmgren** [comment]
Hey @danielminhnguyen! We recently released a version of Ignite that supports symlinks / monorepos. Take a look! #1847

**Reply by danielminhnguyen** [reply]
This is amazing, I will check it out. Thank you!

**Comment by flexbox** [comment]
Hey @danielminhnguyen 
I don't know if it will help you but theses days a lot of solutions are available 

I wrote some of them here
https://github.com/infinitered/ignite/issues/1981#issuecomment-1212016845
